packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 48.8.0 - March 26, 2020
```

Comment:
Do we want to include the changes from W-7318603 as well?

Reply:
Isn't W-7318603 going to the new deploy-retrieve library?

Reply:
@jag-sfdc is right in that it is part of the library so we don't need to add it to this changelog.

Reply:
Sounds good. Thanks for the feedback Jag and Luis!